Entities/Structures/Components/Passive/Magazine/Magazine.as

```diff
@@ -65,47 +65,45 @@ void GetButtonsFor(CBlob@ this, CBlob@ caller)
 
 	if (this.getDistanceTo(caller) > 16.0f || !this.getShape().isStatic()) return;
 
-	CBlob@ carried = caller.getAttachments().getAttachmentPointByName("PICKUP").getOccupied();
-	if (carried !is null)
-	{
-		if (carried.getRadius() > 3.5f || !carried.canBePutInInventory(this)) return;
-
-		CBlob@ item = this.getInventory().getItem(0);
-		if (item !is null && (item.getName() != carried.getName() || item.getQuantity() == item.maxQuantity)) return;
-
-		CBitStream params;
-		params.write_u16(carried.getNetworkID());
+	CBlob@ carried = caller.getCarriedBlob();
+	bool LOAD = (carried !is null);
 
-		CButton@ button = caller.CreateGenericButton(
-		"$"+carried.getName()+"$",              // icon token
-		Vec2f_zero,                             // button offset
-		this,                                   // button attachment
-		this.getCommandID("load"),              // command id
-		getTranslatedString("Load"),            // description
-		params);                                // cbitstream
+	CBlob@ item = this.getInventory().getItem(0);
 
-		button.radius = 8.0f;
-		button.enableRadius = 22.0f;
+	if (LOAD)
+	{
+		if  (!carried.canBePutInInventory(this) 			// doesn't go in inventories to begin with
+			|| !this.getInventory().canPutItem(carried)  	// does go in inventories but doesn't fit
+			|| (item !is null && (item.getName() != carried.getName() || item.getQuantity() == item.maxQuantity)))
```

Comment:
Fix indentation here

Reply:
How should it be fixed?
It looks lined up like this for me https://i.imgur.com/PUjuIvZ.png
Hence the extra space after if.